paperpress.js

```diff
@@ -117,13 +118,40 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? this._marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var dataAttributes = contentData.attributes
+
+		// Iterate through attributes and merge into the default item
+		for (var k in dataAttributes) {
+			if (dataAttributes.hasOwnProperty(k)) {
```

Comment:
But if we check against `item.hasOwnProperty(k)` shouldn't be the same? since item at this point only can have `title`, `slug`, `content` and `path`.
I'm saying this because the coverall mark a `else path not taken` and if we put `item. hasOwnProperty(k)` the else path is taken and then check all the Branches coverall.